### PR TITLE
Add HTSSOP-28-1EP_4.4x9.7mm_P0.65mm_EP2.85x5.4mm

### PR DESCRIPTION
**Pull request by chschlue**
FP PR: https://github.com/KiCad/kicad-footprints/pull/1788

### CODE REVIEW

File: scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/htssop.yaml [hunk 1]
```diff
@@ -199,6 +199,41 @@ HTSSOP-24-1EP_4.4x7.8mm_P0.65mm_EP3.4x7.8mm_Mask2.4x4.68mm_ThermalVias:
     # bottom_pad_size:
     paste_avoid_via: False
 
+HTSSOP-28-1EP_4.4x9.7mm_P0.65mm_EP2.85x5.4mm:
```
Comment (poeschlr) [inline scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/htssop.yaml]: You do not need to duplicate the non thermal vias version. (if thermal vias section is given then non thermal via version is generated as well)

**Comment by poeschlr** [comment]
Dimensions look ok. But you have them duplicated which is not necessary.

File: scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/htssop.yaml (hunk 1 above)
Comment (chschlue) [reply scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/htssop.yaml]: Done

**Comment by poeschlr** [comment]
I also just noticed that all other parts use tab-width = 2 spaces while you used way more. Could you reduce the number of spaces per tab used to align with the rest of the file?

**Comment by chschlue** [comment]
I'd rather use the "one-tab-per-tab" convention but that's apparently frowned upon by Python folks.

Here you go.

**Comment by poeschlr** [comment]
thanks

**Pull request merged**